api/src/main/java/pl/kolendateam/dadcard/characterCard/entity/Character.java

```diff
@@ -65,8 +66,8 @@ public class Character {
     @JdbcTypeCode(SqlTypes.JSON)
     ArrayList<ClassPc> classPcArray;
 
-    int ecl;
-    int levelAdjustment;
+    short ecl;
```

Comment:
co to ec1? Stosowanie skrótów moze byc uznane za nie ladny kod

Reply:
zmieniłem na EffectiveCharacterLv/effectiveCharacterLv